packages/terra-site/src/examples/search-field/SearchFieldBasic.jsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 import SearchField from 'terra-search-field';
 
-const view = () => (
-  <div>
-    <SearchField />
-  </div>
-);
+class SearchFieldView extends React.Component {
 
-export default view;
+  constructor(props) {
+    super(props);
+    this.state = { searchText: '' };
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Search Text: {this.state.searchText}</p>
+        <SearchField onSearch={(searchText) => { this.setState({ searchText }); }} />
```

Comment:
There is a default min search length character of 2. I updated these examples to display a message indicating that until a valid search length is reached, then it will display the search text.